schemas/README-EDDN-schemas.md

```diff
@@ -176,7 +176,10 @@ Each `message` object must have, at bare minimum:
    technically allows for any timezone to be cited you SHOULD provide this in
    UTC, aka 'Zulu Time' as in the example above. You MUST ensure that you are
    doing this properly. Do not claim 'Z' whilst actually using a local time
-   that is offset from UTC.
+   that is offset from UTC. In some schemas several events are batched inside 
```

Comment:
I think it is probably worth saying that there should be a top-level message->timestamp value as well, so as to not trip up listeners' generic code.

Its value should be when the sending software sent the message.  I'm basing this on the proposed algorithm for senders.  We definitely don't want the first event timestamp, but given the possibility of sending being triggered by "10 seconds since last FSSSignalDiscovered event", we don't want it to be based on the latest either.  Sending clients might have other factors in play as well, so the timestamp should be when they *first* attempted sending the message (not later if they retry for any reason).

Reply:
Do you think it should be `SHOULD` or `MUST`?